### ISSUE TITLE
Auto-correct should correct Lint/EndAlignment

### ISSUE THREAD

**Issue by vfonic** [bbatsov/rubocop#3900]
If you run `rubocop -a` on this file:

```ruby
class Klass
  def method_name
    if this
      that
      else
      that_other
      end
  end
  end
```

You will get this output:
```ruby
class Klass
  def method_name
    if this
      that
    else
      that_other
      end
  end
  end
```

Basically, only `else` keyword corrected. If it works for `else` statement, similar logic could be applied to fix `end` keyword.

--------

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.46.0 (using Parser 2.3.3.1, running on ruby 2.3.3 x86_64-darwin16)
```

**Comment by backus** [comment]
Try setting this in your config

```
Lint/EndAlignment:
  AutoCorrect: false
```

**Comment by vfonic** [comment]
Wow, thanks! That works! Do you think it should be ON by default?

**Comment by Drenmi** [comment]
@vfonic: There's an explanation to why it is disabled by default [here](https://github.com/bbatsov/rubocop/pull/1789#issuecomment-92308357). 😊
